.github/workflows/wallet-ext-prs.yml

```diff
@@ -33,9 +33,6 @@ jobs:
       - name: Lint
         working-directory: ${{env.working-directory}}
         run: pnpm lint
-      - name: Test
-        working-directory: ${{env.working-directory}}
-        run: pnpm test
```

Comment:
this PR moves the mnemonics check to sdk/typescript and wallet/ do not have any tests to run

Reply:
Can we still keep this so that it's easy to add unit tests for the wallet?

Reply:
```
 DEV  v0.22.1 /Users/joy/mysten/sui/apps/wallet

include: **/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}
exclude:  **/node_modules/**, **/dist/**, **/cypress/**, **/.{idea,git,cache,output,temp}/**
watch exclude:  **/node_modules/**, **/dist/**

No test files found, exiting with code 1
 ELIFECYCLE  Test failed. See above for more details.
```

added `--recursive` to not make this fail on empty